{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/core/permissions.py

```diff
@@ -1,8 +1,8 @@
 from rest_framework import permissions
 
 
-class CreateOnlyPermissions(permissions.BasePermission):
-    def has_permission(self, request, view):
-        if view.action == "create":
-            return True
-        return False
+class HasUserPermissions(permissions.BasePermission):
```

Comment:
Q, is there a situation where `request.user` might be None? if so wont that cause a 500 error crash

Reply:
Should not happen ever

Reply:
Notice that the viewset where this is used also handles user creation requests. There are automated tests that cover that case.

I do think I've had to handle some sort of none case in the past, but I haven't seen that in a long while. But should that bug arise it would be a small/quick/obvious fix

Reply:
I think the Authentication Middleware always adds `user` to the request. If the client is unauthenticated, then it sets `request.user` to an instance of `AnonymousUser`. But as Ed said, it should never be `None` or falsey. Could only happen if that middleware is removed.

Reply:
That tracks. Otherwise `is_anonymous` wouldn't be very useful. That might have been a bug in older versions that people reported

Reply:
ah nvm this is used on the object level!